packages/create-mechanic/function-examples/adaptive-grid/dependencies.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@mechanic-design/engine-react": "^2.0.0-beta.10"
```

Comment:
Do we need to add `@mechanic-design/ui-components` here as well?

I tested this locally by adding this function to the DSI Logo maker and had to install `@mechanic-design/ui-components` manually, as the app would crash otherwise.

Reply:
Great catch! I replicated it and go to the error too. Should be fixed now.